ocs_ci/ocs/node.py

```diff
@@ -393,3 +394,49 @@ def node_network_failure(node_names, wait=True):
             node_names=node_names, status=constants.NODE_NOT_READY
         )
     return True
+
+
+def get_osd_running_nodes():
+    """
+    Gets the osd running node names
+
+    Returns:
+        list: OSD node names
+
+    """
+    return [
+        pod.get_pod_node(osd_node).name for osd_node in pod.get_osd_pods()
+    ]
+
+
+def get_app_pod_running_nodes(pod_obj):
+    """
+    Gets the app pod running node names
+
+    Args:
+        pod_obj (list): List of app pod objects
+
+    Returns:
+        list: App pod running node names
+
+    """
+    return [pod.get_pod_node(node).name for node in pod_obj]
```

Comment:
```suggestion
    return [pod.get_pod_node(obj_pod).name for obj_pod in pod_obj]
```
obj_pod or any other meaningful name.